probe_src/python/probe_py/manual/cli.py

```diff
@@ -169,5 +180,40 @@ def dump(
                     print(pid, exid, tid, op_no, op.data)
                 print()
 
+@app.command()
+def export(
+    input: pathlib.Path = pathlib.Path("dataflow_graph.pkl"),
+    makefile : bool = typer.Option(default=False, help="export in makefile format"),
+    nextflow : bool = typer.Option(default=True, help="export in nextflow format"),
```

Comment:
Let's give the user an enum of options instead of a one-hot encoding to select the output format. <https://typer.tiangolo.com/tutorial/parameter-types/enum/>